css/search.scss

```diff
@@ -1,8 +1,8 @@
 @import "../css/variables";
 
-.search-container {
-    display: none;
-}
+// .search-container {
```

Comment:
if this is not needed and you don't anticipate bringing it back, please remove this block.